its/src/test/kotlin/org/sonarlint/intellij/its/utils/SettingsUtils.kt

```diff
@@ -70,7 +70,8 @@ class SettingsUtils {
                 searchRule(ruleKey)
                 Pause.pause(1000)
                 findText(ruleText).doubleClick()
-                button("Apply").clickWhenEnabled()
+                Pause.pause(10000)
```

Comment:
Perhaps you could do something dynamic, waitFor condition based on button("Apply") being enabled?